crates/ruff_workspace/src/options.rs

```diff
@@ -2942,28 +2942,30 @@ pub struct FormatOptions {
     )]
     pub indent_style: Option<IndentStyle>,
 
-    /// Configures the preferred quote character for strings. Valid options are:
-    ///
+    /// Configures the preferred quote character for strings. The recommended options are
     /// * `double` (default): Use double quotes `"`
     /// * `single`: Use single quotes `'`
-    /// * `preserve` (preview only): Keeps the existing quote character. We don't recommend using this option except for projects
-    ///    that already use a mixture of single and double quotes and can't migrate to using double or single quotes.
     ///
     /// In compliance with [PEP 8](https://peps.python.org/pep-0008/) and [PEP 257](https://peps.python.org/pep-0257/),
-    /// Ruff prefers double quotes for multiline strings and docstrings, regardless of the
+    /// Ruff prefers double quotes for triple quoted strings and docstrings, regardless of the
     /// configured quote style.
     ///
-    /// Ruff may also deviate from using the configured quotes if doing so requires
-    /// escaping quote characters within the string. For example, given:
+    /// Ruff deviates from using the configured quotes if doing so prevents the need for
+    /// escaping quote characters inside the string:
     ///
     /// ```python
     /// a = "a string without any quotes"
     /// b = "It's monday morning"
     /// ```
     ///
-    /// Ruff will change `a` to use single quotes when using `quote-style = "single"`. However,
-    /// `b` remains unchanged, as converting to single quotes requires escaping the inner `'`,
-    /// which leads to less readable code: `'It\'s monday morning'`. This does not apply when using `preserve`.
+    /// Ruff will change the quotes of the string assigned to `a` to single quotes when using `quote-style = "single"`.
+    /// However, ruff uses double quotes for he string assigned to `b` because using single quotes would require escaping the `'`,
+    /// which leads to the less readable code: `'It\'s monday morning'`.
+    ///
+    /// In addition, Ruff supports the quote style `preserve` for projects that already use
+    /// a mixture of single and double quotes and can't migrate to the `double` or `single` style.
+    /// The quote style `preserve` leaves the quotes of all strings unchanged.
+    /// We don't recommend using `preserve` for new project because it doesn't enforce consistent quotes.
```

Comment:
I'd probably omit this last sentence (it's implied in the previous sentences that this is intended for existing projects). (If it stays, "project" should be "projects".)